v2/examples/spiffe-jwt-using-proxy/README.md

```diff
@@ -0,0 +1,224 @@
+	.
+	.
+	.
+	req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", svid.Marshal()))
+```
+
+The **server workload** uses the `JWTSource` to authenticate the client by calling the [ParseAndValidate](https://pkg.go.dev/github.com/spiffe/go-spiffe/v2/svid/jwtsvid?tab=doc#ParseAndValidate) function:
+```go
+	_, err := jwtsvid.ParseAndValidate(token, a.jwtSource, a.audiences)
+```
+Where:
+- token is the marshalled JWT-SVID sent by the client in the `Authorization` header.
+- a.jwtSource is the `JWTSource`.
+- a.audiences is a slice of strings. Specifies a list of expected audiences in the `aud` field of the token.
+
+When `ParseAndValidate` returns an error, the server returns an `Unauthorized` status. Otherwise, the request continues normal processing.
+
+## That is it!
+As we can see the go-spiffe library allows your application avoiding to deal with the implementation details of the Workload API. You just create the SVID sources you need and then ask the library for what you need in an ergonomic fashion.
+
+## Building
+To build the client workload:
+```bash
+cd examples/spiffe-jwt-using-proxy/client
+go build
+```
+
+To build the proxy workload:
+```bash
+cd examples/spiffe-jwt-using-proxy/proxy
+go build
+```
+
+To build the server workload:
+```bash
+cd examples/spiffe-jwt-using-proxy/server
+go build
+```
+
+## Running
+This example assumes the following preconditions:
+- There is a SPIRE server and agent up and running.
+- There is a Unix workload attestor configured.
+- The trust domain is `example.org`.
+- The agent SPIFFE ID is `spiffe://example.org/host`.
+- There is a `server-workload` and `client-workload` user in the system.
+
+### 1. Create the registration entries
+Create two registration entries, one for the client workload and another for the server and proxy workloads:
+
+Server:
+```bash
+./spire-server entry create -spiffeID spiffe://example.org/server \
+                            -parentID spiffe://example.org/host \
+                            -selector unix:user:server-workload
+```
+
+Client: 
+```bash
+./spire-server entry create -spiffeID spiffe://example.org/client \
+                            -parentID spiffe://example.org/host \
+                            -selector unix:user:client-workload
+```
+
+We will use the `server-workload` user to run the proxy because we want it to be as transparent as possible to the client. By using this user, the proxy will get an SVID with the same SPIFFE ID as the server, due to the `unix:user:server-workload` selector used when registering the entry.
+
+### 2. Start the server
+Start the server with the `server-workload` user:
+```bash
+sudo -u server-workload ./server
+```
+
+### 3. Start the proxy
+Start the proxy with the `server-workload` user:
+```bash
+sudo -u server-workload ./proxy
+```
+
+### 4. Run the client
+Run the client with the `client-workload` user:
+```bash
+sudo -u client-workload ./client
+```
+
+For each componet the logs would contain:  
```

Comment:
```suggestion
For each component the logs would contain:  
```
Could we list the location(s) of the log files?

Reply:
Yeah sure. They log to standard out, so I'll clarify this.